File: Java files/src/main/java/com/cts/entities/Priority.java
package com.cts.entities;

public class Priority {

	private int priorityId;
	private String priorityName;

	public Priority() {

		priorityId = 0;
		priorityName = "";
	}

	public int getPriorityId() {

		return priorityId;
	}

	public void setPriorityId(int priorityId) {

		this.priorityId = priorityId;
	}

	public String getPriorityName() {

		return priorityName;
	}

	public void setPriorityName(String priorityName) {

		this.priorityName = priorityName;
	}

}

File: Java files/src/main/webapp/resources/libs/models/userMenuModels/priority.js
/*! CTS Priority Model */

var Priority = Backbone.Model.extend({ 
	
	defaults: {
		priorityId: '',
		priorityName: ''
	},
	
	check: function() {
		
		var errors = {};
		
		/*if (this.get('priorityId') == '') {
			errors['#priorityId'] = getValue('mandatory');
			errorsFound = true;
		};*/
		
		if (this.get('priorityName') == '') {
			errors['#ticketPriorityDropbox'] = getValue('mandatory');
		} else {
			errors['#ticketPriorityDropbox'] = '';
		};
		
		return errors;
	}
});